packages/arb-token-bridge-ui/tests/e2e/specs/depositERC20.cy.ts

```diff
@@ -8,169 +8,186 @@ import {
   getL1NetworkConfig,
   zeroToLessThanOneETH,
   getL1NetworkName,
-  getL2NetworkName
+  getL2NetworkName,
+  ERC20TokenSymbol
 } from '../../support/common'
 
+const { _ } = Cypress
+
 const moreThanZeroBalance = /0(\.\d+)/
 
-describe('Deposit ERC20 Token', () => {
+const depositTestCases = {
+  'Standard ERC20': {
+    symbol: ERC20TokenSymbol,
+    l1Address: Cypress.env('ERC20_TOKEN_ADDRESS_PARENTCHAIN'),
+    l2Address: Cypress.env('ERC20_TOKEN_ADDRESS_CHILDCHAIN')
+  },
+  WETH: {
+    symbol: 'WETH',
+    l1Address: Cypress.env('L1_WETH_ADDRESS'),
+    l2Address: Cypress.env('L2_WETH_ADDRESS')
+  }
+}
+
+describe('Deposit Token', () => {
   // when all of our tests need to run in a logged-in state
   // we have to make sure we preserve a healthy LocalStorage state
   // because it is cleared between each `it` cypress test
 
   const isOrbitTest = Cypress.env('ORBIT_TEST') == '1'
   const depositTime = isOrbitTest ? 'Less than a minute' : '9 minutes'
-  const l1WethAddress = Cypress.env('L1_WETH_ADDRESS')
 
   // Happy Path
-  context('User has some ERC20 and is on L1', () => {
-    let l1ERC20bal: string
-
-    // log in to metamask before deposit
-    beforeEach(() => {
-      getInitialERC20Balance({
-        tokenAddress: l1WethAddress,
-        multiCallerAddress: getL1NetworkConfig().multiCall,
-        address: Cypress.env('ADDRESS'),
-        rpcURL: Cypress.env('ETH_RPC_URL')
-      }).then(val => (l1ERC20bal = formatAmount(val)))
-    })
+  _.each(depositTestCases, (testCase, tokenType) => {
+    context(`User has some ${tokenType} and is on L1`, () => {
+      let l1ERC20bal: string
+
+      // log in to metamask before deposit
```

Comment:
Majority of these diffs in `depositErc20.cy.ts` and `withdrawErc20.cy.ts` are due to indentation after putting these tests inside an `each` loop for different test cases.

Reply:
The only update in these files is replacing hardcoded `WETH` with the current token of the iteration.